schema/crdb/dbinit.sql

```diff
@@ -1107,6 +1107,15 @@ CREATE TABLE IF NOT EXISTS omicron.public.instance (
      */
     auto_restart_policy omicron.public.instance_auto_restart,
 
+    /*
+     * Which disk, if any, is the one this instance should be directed to boot
+     * from. With a boot device selected, guest OSes cannot configure their
+     * boot policy for future boots, so also permit NULL to indicate a guest
+     * does not want our policy, and instead should be permitted control over
+     * its boot-time fates.
+     */
+    boot_device UUID,
+
```

Comment:
ID columns tend to end with `_id`

Reply:
oh, they sure do

Reply:
(https://github.com/oxidecomputer/omicron/pull/6585/commits/da9e29c8eae5f6ee54970c8c9f47669b6f892d6d)